src/main/scala/beam/utils/scenario/BeamScenarioLoader.scala

```diff
@@ -150,7 +150,7 @@ class BeamScenarioLoader(
       personAttributes.putAttribute(personId, "sex", sexChar)
       person.getAttributes.putAttribute("sex", sexChar)
       person.getAttributes.putAttribute("age", personInfo.age)
-
+      person.getAttributes.putAttribute("industry", personInfo.industry)
```

Comment:
`personInfo.industry` is an option, please use `getOrElse` otherwise the saved value will be `Some("IndustryType")`

Reply:
applied changes as per suggestion